app/src/components/Statistics/Statistics.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { StatisticsElement } from "./StatisticsElement";
+import brandIcon from "../../images/icon-brand-recognition.svg";
+import detailedIcon from "../../images/icon-detailed-records.svg";
+import fullyIcon from "../../images/icon-fully-customizable.svg";
+import { StyledWrapper, StyledHeader, StyledHeading, StyledDescription, StyledStatiscticsElements } from "./Statistics.styles";
+
+export const Statistics = () => {
+  const obj = [
+    {
+      title: "Brand Recognition",
+      description:
+        "Boost your brand recognition with each click. Generic links don't mean a thing. Branded links help instil confidence in your content",
+      icon: brandIcon
+      },
+    {
+      title: "Detailed Records",
+      description: "Gain insights into who is clicking",
+      icon: detailedIcon
+    },
+    {
+      title: "Fully Customizable",
+      description: "Improve brand awareness",
+      icon: fullyIcon
+    },
+  ];
+
+  return (
+    <StyledWrapper>
+      <StyledHeader>
+        <StyledHeading>Advanced Statistics</StyledHeading>
+        <StyledDescription>
+          Track how your links are performing across the web with our advanced
+          statistics dashboard
+        </StyledDescription>
+      </StyledHeader>
+      <StyledStatiscticsElements>
+        {obj.map((singleObj) => {
```

Comment:
I'm 100% sure that if you open the console in the browser then You will see the red error stating that "key" should be provided while rendering the element in the loop. Please fix that ;)

Reply:
https://reactjs.org/docs/lists-and-keys.html

Reply:
also, You can destructure the values 

`{obj.map(({icon,title,description}) => {`